diff --git a/toolchain/BUILD.llvm_repo.tpl b/toolchain/BUILD.llvm_repo.tpl
index 0cd8b11..20162cc 100644
--- a/toolchain/BUILD.llvm_repo.tpl
+++ b/toolchain/BUILD.llvm_repo.tpl
@@ -62,5 +62,11 @@
 cc_library(
     name = "clang_headers",
-    hdrs = glob(["include/**"]),
+    hdrs = glob(
+        ["include/**"],
+        exclude = [
+            "include/c++/**",
+            "include/*/c++/**",
+        ],
+    ),
     strip_include_prefix = "include",
 )
